### PR TITLE
OCPBUGS-39217: Add yq v4 to ci image

### PR DESCRIPTION
**Pull request by vr4manta**
OCPBUGS-39217

### Changes
- Add yq to ci image

### CODE REVIEW

**Comment by vr4manta** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-ovn-upi

**Comment by sdodson** [comment]
/hold
Clear whenever you're happy with those test results

File: images/installer/Dockerfile.upi.ci.rhel8 [hunk 1]
```diff
@@ -43,6 +45,9 @@ RUN yum update -y && \
     rm -rf /var/cache/yum/* && \
     chmod g+w /etc/passwd
 
+COPY --from=yq3 /yq /bin/yq-go
+COPY --from=yq4 /usr/bin/yq /bin/yq-v4
+
```
Comment (r4f4) [inline images/installer/Dockerfile.upi.ci.rhel8:50]: We want to remove the curl from the yq below otherwise you're overriding the binary you just copied.

Comment (vr4manta) [reply images/installer/Dockerfile.upi.ci.rhel8:50]: ah good catch.  i'll update that.

**Comment by vr4manta** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.13-e2e-vsphere-ovn-upi

**Comment by sdodson** [comment]
Doesn't look like the test jobs are using the image built with this PR. I imagine something somewhere selects the latest accepted image instead. I'm fine moving this forward but given that latching we may not be able to get a 4.13 build accepted without forcing it, since the vSphere installs will still fail unless they properly update to this image before its accepted.

**Comment by vr4manta** [comment]
> Doesn't look like the test jobs are using the image built with this PR. I imagine something somewhere selects the latest accepted image instead. I'm fine moving this forward but given that latching we may not be able to get a 4.13 build accepted without forcing it, since the vSphere installs will still fail unless they properly update to this image before its accepted.

I am still looking into this.  I assumed that the payload test would pull it in.  I'm reaching out to others to see what I might be missing.

**Comment by r4f4** [comment]
> Doesn't look like the test jobs are using the image built with this PR. I imagine something somewhere selects the latest accepted image instead. I'm fine moving this forward but given that latching we may not be able to get a 4.13 build accepted without forcing it, since the vSphere installs will still fail unless they properly update to this image before its accepted.

The payload test should use the built image unless the CI steps are forcing a version of `upi-installer` different from the current branch afaik.

**Comment by vr4manta** [comment]
From what we can tell, running payload test doesn't use the image built in this PR.  But when you look at the upi job that was kicked off automatically, it is targeting 4.13 and it has the changes and does not fail.  I am ok with merging this PR and monitoring it again.

**Comment by vr4manta** [comment]
/unhold
I am happy w/ the upi (non payload) tests.

**Comment by sdodson** [comment]
/override ci/prow/e2e-azure-ovn-upi
100% failure rate over entire history of this test

**Pull request merged**